### PR TITLE
Add GeneFlow workflow instructions to docs

### PR DESCRIPTION
**Pull request by jhphan**

### CODE REVIEW

File: docs/source/sections/gcp-info/GeneFlow-RNAseq.rst [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+	    map:
+	      uri: ${sort->output}
+	      regex: (.*).bam
+	    template:
+	      bam: ${sort->output}/${1}.bam
+	      gtf: ${workflow->gtf}
+	      output: ${1}
+	...
+
+The **inputs** and **parameters** blocks define the inputs and paramaters that need to be passed to the workflow upon execution. Some of these inputs and parameters are optional or have default values.
```
Comment (DeenaBleich) [inline docs/source/sections/gcp-info/GeneFlow-RNAseq.rst:173]: 'paramaters' is a typo and should be 'parameters'

File: docs/source/sections/gcp-info/GeneFlow-RNAseq.rst [hunk 2]
```diff
@@ -0,0 +1,225 @@
+========================
+Running GeneFlow RNA-seq
+========================
+
+This GeneFlow `RNA-seq <https://www.technologynetworks.com/genomics/articles/rna-seq-basics-applications-and-protocol-299461#:~:text=RNA%2Dseq%20(RNA%2Dsequencing,patterns%20encoded%20within%20our%20RNA.>`_ workflow maps read-pairs to a reference genome and produces a transcript. 
+
+`GeneFlow <https://github.com/CDCgov/geneflow2>`_ enables modular and reproducible scientific workflows by leveraging re-usable, containerized steps. Custom workflow steps can be implemented using either Docker or Singularity containers. Additional documentation for GeneFlow can be found `here <https://geneflow.gitlab.io>`_.
```
Comment (DeenaBleich) [inline docs/source/sections/gcp-info/GeneFlow-RNAseq.rst:7]: Change 're-usable' to 'reusable'. Probably could be written either way, but it is 'reusable' later, so it is good to be consistent.

File: docs/source/sections/gcp-info/GeneFlow-RNAseq.rst [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+	    │       ├── sample_final_transcript.gtf
+	    │       └── t_data.ctab
+	    └── sort
+		├── _log
+		│   ├── gf-0-sort-sample-bam.err
+		│   ├── gf-0-sort-sample-bam.out
+		│   └── sample.bam-samtools-sort.stderr
+		└── sample.bam
+
+The script will run Docker containers for `hisat2 <http://daehwankimlab.github.io/hisat2/>`_ , `samtools <http://www.htslib.org/>`_, and `stringtie <https://ccb.jhu.edu/software/stringtie/>`_ to do the work. **sample.bam** contains the sequence alignment data produced by mapping reads to the reference genome **sample.bam**. Additional information about gtf and tsv outputs of stringtie can be found `here <http://ccb.jhu.edu/software/stringtie/index.shtml?t=manual>`_. The **sample_final_transcript.gtf** contains details of the transcripts that StringTie assembles from RNA-Seq data, while **sample.tsv** contains gene abundances.
```
Comment (DeenaBleich) [inline docs/source/sections/gcp-info/GeneFlow-RNAseq.rst:223]: Remove space before first comma

File: docs/source/sections/gcp-info/GeneFlow-RNAseq.rst [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+
```
Comment (DeenaBleich) [inline docs/source/sections/gcp-info/GeneFlow-RNAseq.rst:224]: There doesn't appear to be any new results in here.

File: docs/source/sections/gcp-info/GeneFlow-RNAseq.rst (hunk 1 above)
Comment (jhphan) [reply docs/source/sections/gcp-info/GeneFlow-RNAseq.rst]: Fixed

File: docs/source/sections/gcp-info/GeneFlow-RNAseq.rst (hunk 2 above)
Comment (jhphan) [reply docs/source/sections/gcp-info/GeneFlow-RNAseq.rst]: Fixed

File: docs/source/sections/gcp-info/GeneFlow-RNAseq.rst (hunk 3 above)
Comment (jhphan) [reply docs/source/sections/gcp-info/GeneFlow-RNAseq.rst]: Fixed

File: docs/source/sections/gcp-info/GeneFlow-RNAseq.rst (hunk 4 above)
Comment (jhphan) [reply docs/source/sections/gcp-info/GeneFlow-RNAseq.rst:224]: The main result files should be the same as the other RNAseq workflows.

**Pull request closed without merging**